capsules/src/alarm.rs

```diff
@@ -147,6 +150,18 @@ impl<A: Alarm<'a>> Driver for AlarmDriver<'a, A> {
                         td.expiration = Expiration::Abs(time as u32);
                         (ReturnCode::SuccessWithValue { value: time }, true)
                     },
+                    5 /* Set relative expiration */ => {
+                        let mut time = now.wrapping_add (data as u32) as usize;
+                        if time.wrapping_sub (now as usize) <= MIN_TICS {
```

Comment:
We probably want `MIN_TICS` to in absolute time and converted to the frequency units of the clock

Reply:
A comment here explaining why this padding is added would be useful.

Reply:
MIN_TICS was experimental at 16 KHz, I added comments and computed MIN_TICS based on the timer frequency.